.github/build-xcframework.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# CREDIT: https://forums.swift.org/t/how-to-build-swift-package-as-xcframework/41414/4
+# NOTE: This does modify the Package.swift file, which is less than ideal.
+set -x
+set -e
+
+# Pass scheme name as the first argument to the script
+NAME=$1
+
+# Build the scheme for all platforms that we plan to support
+for PLATFORM in "${@:2}"; do
+
+  case $PLATFORM in
+  "iOS")
+    RELEASE_FOLDER="Release-iphoneos"
+    ;;
+  "iOS Simulator")
+    RELEASE_FOLDER="Release-iphonesimulator"
+    ;;
+  "WatchOS")
+    RELEASE_FOLDER="Release-watchos"
+    ;;
+  "WatchOS Simulator")
+    RELEASE_FOLDER="Release-watchossimulator"
+    ;;
+  "TvOS")
+    RELEASE_FOLDER="Release-tvos"
+    ;;
+  "TvOS Simulator")
+    RELEASE_FOLDER="Release-tvossimulator"
+    ;;
+  "macOS")
+    RELEASE_FOLDER="Release-macos"
```

Comment:
I tried to pull all 3 libraries into a Mac Catalyst app, and they all failed because `While building for Mac Catalyst, no library for this platform was found in...`

Reply:
Is this something you'd like me to fix before you'll consider merging or is this informational and something we'll need to add in as a follow-up?

Reply:
I'm checking to see if it's something we can tweak right now or if it will need to be a follow-up.  Leaving the comment to prevent merging until we decide which way to go.

Reply:
I think we'll wait until a request comes in for Catalyst in the hosted frameworks.  We know we support it using SPM, and this is just meant to be an additional option, not the primary form of consuming our library.